source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/input/TextInputRenderer.java

```diff
@@ -162,7 +270,14 @@ public void afterTextChanged(Editable s)
         }
         else
         {
-            viewGroup.addView(editText);
+            if(textInputViewGroup != null)
+            {
+                viewGroup.addView(textInputViewGroup);
```

Comment:
>viewGroup.addView(textInputViewGroup); [](start = 16, length = 38)

can you add a test where you have a text input with inline action and give it height stretch?? Something similar to ColumnSet.Input.Text.VerticalStretch.json

Reply:
will add it as part of new changes to UWP since UWP needs a fix for vertical alignment

---
In reply to: [234315972](https://github.com/Microsoft/AdaptiveCards/pull/2167#discussion_r234315972) [](ancestors = 234315972)